src/modules/dashboard/sagas/actions/stakeMotion.ts

```diff
@@ -0,0 +1,116 @@
+import { call, fork, put, takeEvery } from 'redux-saga/effects';
+import { ClientType, VotingReputationClient } from '@colony/colony-js';
```

Comment:
There's not such export in colony-js, the export you're looking for `ExtensionClient`

Reply:
I'll change it to `ExtensionClient`, but how come there's a `VotingReputationClient` export available? I'm using `4.0.0-rc.0`

Reply:
There isn't. That's why your CI doesn't pass, and that's why you get this error:

![Screenshot from 2021-04-16 16-01-15](https://user-images.githubusercontent.com/1193222/115027974-06881d00-9ecd-11eb-95ec-c85551646f5f.png)

Reply:
Mmmm, I just saw that the VotingReputationClient it's in the `.d.ts` of the index of colonyJS. That's why it was letting me.

Either way, fixed.